test/uk/gov/hmrc/agentservicesaccount/controllers/ManageAgentServicesAccountLandingSpec.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentservicesaccount.controllers
+
+
+import org.jsoup.Jsoup
+import play.api.i18n.{Lang, MessagesApi}
+import play.api.test.Helpers._
+import play.api.test.FakeRequest
+import uk.gov.hmrc.agentmtdidentifiers.model.{Arn, OptedInReady}
+import uk.gov.hmrc.agentservicesaccount.models.AccessGroupSummaries
+import uk.gov.hmrc.agentservicesaccount.stubs.AgentPermissionsStubs.{givenAccessGroupsForArn, givenArnAllowedOk, givenOptinStatusSuccessReturnsForArn, givenSyncEacdSuccess}
+import uk.gov.hmrc.agentservicesaccount.support.Css.{H1, paragraphs}
+import uk.gov.hmrc.agentservicesaccount.support.{BaseISpec, Css}
+import uk.gov.hmrc.http.SessionKeys
+import uk.gov.hmrc.auth.core.{Enrolment, EnrolmentIdentifier}
+
+
+
+
+
+
+class ManageAgentServicesAccountLandingSpec extends BaseISpec {
+
+
+  implicit val lang: Lang = Lang("en")
+  implicit val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
+
+
+  val arn = "TARN0000001"
+  val agentEnrolment: Enrolment = Enrolment("HMRC-AS-AGENT", Seq(EnrolmentIdentifier("AgentReferenceNumber", arn)), state = "Activated", delegatedAuthRule = None)
+
+
+
+  val controller: ManageAgentServicesAccountLanding = app.injector.instanceOf[ManageAgentServicesAccountLanding]
+
+  "showAccessGroupSummaryForASA" should  {
+
+    val ASAAccountTitle = "Manage access in the agent services account - Agent services account - GOV.UK"
+
+    "return Status: OK containing correct content" in {
+
+
+      givenAuthorisedAsAgentWith(arn)
+      val response = await(controller.showAccessGroupSummaryForASA(FakeRequest("GET", "/agent-services-access").withSession(SessionKeys.authToken -> "Bearer XYZ"))) //URL response created to mock webpage
+
+
+      status(response) shouldBe OK // I except response to be OK(200)
+
+      val html = Jsoup.parse(contentAsString(response))
+
+      html.title() shouldBe ASAAccountTitle
+
+
+    }
```

Comment:
This test case is redundant, it's covering the same as "return page correct content when OptOut" - well apart from the the title check, which you should add to both opt-in and opt-out cases :D